### DISCUSSION TITLE
WIP Wifi: quiet.py

### DISCUSSION THREAD

**Discussion by JarbasAl** [OpenVoiceOS/ovos-buildroot#144] [Show and tell]
needs https://github.com/OpenVoiceOS/ovos-core/pull/163 for decoder using https://github.com/OpenVoiceOS/quiet.py

companion website: https://github.com/OpenVoiceOS/hey-wifi

**Comment by JarbasAl** [comment]
@j1nx and @AIIX would be great if you could start including quiet.py in your images

**Comment by j1nx** [comment]
Have put it on the buildroot todo list. Need to get that list online as part of the goals and roadmaps for buildroot.
